examples/low-level-rustls/src/main.rs

```diff
@@ -30,8 +37,8 @@ async fn main() {
         .init();
 
     let rustls_config = rustls_server_config(
-        "examples/tls-rustls/self_signed_certs/key.pem",
-        "examples/tls-rustls/self_signed_certs/cert.pem",
+        PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("self_signed_certs/key.pem"),
```

Comment:
Maybe use platform-native path separators while you're at it?

```suggestion
        PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("self_signed_certs").join("key.pem"),
```

Reply:
Oh whoops sorry. I'll fix those!

Reply:
There we go!